frontend/utilities/campaign_helpers/index.ts

```diff
@@ -114,13 +122,25 @@ const updateCampaignStateFromStatus = (
 
 export const updateCampaignState = (socketData: ISocketData) => {
   return ({ campaign }: ICampaignState) => {
+    const { renderFlash } = useContext(NotificationContext);
     switch (socketData.type) {
       case "totals":
         return updateCampaignStateFromTotals(campaign, socketData);
       case "result":
         return updateCampaignStateFromResults(campaign, socketData);
       case "status":
         return updateCampaignStateFromStatus(campaign, socketData);
+      case "error":
+        if (socketData.data.includes("unexpected exit in receiveMessages")) {
+          const campaignID = socketData.data.substring(
+            socketData.data.indexOf("=") + 1
+          );
+          renderFlash(
```

Comment:
I like that this error flash message will render for both live queries and live policies.